doc/api/n-api.md

```diff
@@ -1325,7 +1354,7 @@ napi_status napi_create_string_utf8(napi_env env,
 ```
 
 - `[in] env`: The environment that the API is invoked under.
-- `[in] s`: Character buffer representing a UTF8-encoded string.
+- `[in] str`: Character buffer representing a UTF8-encoded string.
 - `[in] length`: The length of the string in characters, or -1 if it is
```

Comment:
Also fix the doc for the `length` parameter of `napi_create_string_utf16`. That length is in _two-byte code units_, not characters. (GitHub won't let me put a comment on that line.)